lib/store.js

```diff
@@ -11,6 +11,8 @@ var defaultStoreOptions = {
   clean: true
 }
 
+var Map = require('es6-map')
```

Comment:
we don't need es6-map, `Map` should be around in all the environments this run. Or maybe not?

Reply:
> It should be noted that a Map which is a map of an object, especially a dictionary of dictionaries, will only map to the object's insertion order. In ES2015 this is ordered for objects but for older versions of ES, this may be random and not ordered.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map#Description 

es6-map can preserve insertion order if ES version is older.

Reply:
Can you please add a comment before the `require('es6-map')` with  that comment? This is the case for IE11 I presume.

Reply:
Sure, I just added it.